getpaid/backends/pagseguro/views.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse, Http404, HttpResponseRedirect
+from django.views.generic import DetailView
+from django.views.generic.base import View
+from getpaid.backends.pagseguro import PaymentProcessor
+from getpaid.models import Payment
+
+logger = logging.getLogger('getpaid.backends.pagseguro')
+
+
+class NotificationsView(View):
+    """
+    This view answers on Moip notifications requests.
+    See http://labs.moip.com.br/referencia/nasp/
+
+    The most important logic of this view is delegated to the
+    ``PaymentProcessor.processNotification()`` method
+    """
+    def post(self, request, *args, **kwargs):
+        try:
+
+            request.encoding = 'ISO-8859-1'
+            dados = dict((k, v.encode('ISO-8859-1')) for k, v in request.POST.items())
+            
```

Comment:
This doesn't work like that. I think you need to get the encoding and use it to decode your post params to unicode. This needs fixing.
https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.encoding
We try to use unicode everywhere in the code.